compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -136,7 +136,7 @@ public fun KtClassOrObject.scope(
     .let { classLiteralExpression ->
       if (classLiteralExpression == null) {
         throw AnvilCompilationException(
-          "The first argument for $annotationFqName must be a class literal: $text",
+          "Couldn't find scope for $annotationFqName.",
```

Comment:
There's a test for this in `ModuleMergerTest`.  This change makes it so that the two `scope` extension functions use the same message.